.release-please-manifest.json

```diff
@@ -1,3 +1,3 @@
 {
-  ".": "0.13.0"
+  ".": "0.13.1"
```

Comment:
should it be 0.14 given we have 3 fixes

Reply:
The are all "fixes", the minor change only happens on features. This is release-please mechanics.